### DISCUSSION TITLE
Specify the cmake.exe directory

### DISCUSSION THREAD

**Discussion by trampas** [microsoft/vscode-cmake-tools#4212] [General]
I have multiple projects and each one requires a specific cmake.exe version.  I was wondering how to specify path for which cmake the tool will use?

**Comment by gcampbell-msft** [comment]
@trampas Thanks for the question! The best way to do this is to use CMakePresets, in each configure preset in a CMakePresets.json file, there is a `cmakeExecutable` field. 

Please let me know if this works for you!
